examples/example_glfw_vulkan/main.cpp

```diff
@@ -467,11 +470,13 @@ int main(int, char**)
         glfwPollEvents();
 
         // Resize swap chain?
-        if (g_SwapChainRebuild && g_SwapChainResizeWidth > 0 && g_SwapChainResizeHeight > 0)
+        if (g_SwapChainRebuild)
         {
             g_SwapChainRebuild = false;
+            int width, height;
+            glfwGetWindowSize(window, &width, &height);
             ImGui_ImplVulkan_SetMinImageCount(g_MinImageCount);
```

Comment:
This changed `glfwGetFramebufferSize()` above to `glfwGetWindowSize()` here, the later is correct and I don't think we have an equivalent in the PlatformIO interface.